index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Hello Microverse</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <div class="logo">My Logo</div>
```

Comment:
- [ ] Kindly change this `My Logo` text into an image (with a link) or simply use an `<a>` tag with its correspondent `href` property. Most web pages use their own logo to go back to the main page, so it would be nice to start practicing doing it.